vaadinfrontend/src/main/java/life/qbic/datamanager/views/general/PageArea.java

```diff
@@ -5,11 +5,11 @@
 import java.io.Serial;
 
 /**
- * <b><class short description - 1 Line!></b>
+ * <b>A specific display area of the app that is neither header nor footer</b>
  *
- * <p><More detailed description - When to use, what it solves, etc.></p>
+ * <p>Contains main content OR support content (side-bar) of the respective page.</p>
  *
- * @since <version tag>
+ * @since <1.0.0>
  */
 @Tag(Tag.DIV)
 public class PageArea extends Component {
```

Comment:
Shouldn't the PageArea Component implement the HasComponents interface So child element can be added?